DirectXTex/DirectXTex.h

```diff
@@ -471,7 +472,7 @@ namespace DirectX
     HRESULT __cdecl GenerateMipMaps( _In_ const Image& baseImage, _In_ DWORD filter, _In_ size_t levels,
                                      _Inout_ ScratchImage& mipChain, _In_ bool allow1D = false );
     HRESULT __cdecl GenerateMipMaps( _In_reads_(nimages) const Image* srcImages, _In_ size_t nimages, _In_ const TexMetadata& metadata,
-                                     _In_ DWORD filter, _In_ size_t levels, _Inout_ ScratchImage& mipChain );
+                                     _In_ DWORD filter, _In_ size_t levels, _Inout_ ScratchImage& mipChain, const float alphaCoverageRatio = 1.0f);
```

Comment:
To avoid clutter I've not been marking simple parameter passed by value as ``const``.